setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
@bjlittle 

this 'name' appears in the folder structure of the installation via setuptools, e.g.:
```
>>> import iris; iris
<module 'iris' from '/home/h04/itmh/.local/lib/python2.7/site-packages/iris_scitools-1.12.0.dev0-py2.7.egg/iris/__init__.pyc'>
```

and in the PKG-INFO
```
Metadata-Version: 1.0
Name: iris-scitools
Version: 1.12.0.dev0
Summary: Data interoperability library
Home-page: http://scitools.org.uk/iris/
Author: UK Met Office
Author-email: https://groups.google.com/forum/#!forum/scitools-iris
License: LGPLv3
Description: Data interoperability library for meteorological and 
        oceanographic data formats, providing 
        data analysis and visualisation capabilities
        
Platform: any

```